src/kernels/jupyter/types.ts

```diff
@@ -254,6 +254,10 @@ export interface IJupyterServerUriEntry {
      * An optional display name to show for this server as opposed to just the Uri
      */
     displayName?: string;
+    /**
+     * Whether the server is validated by its provider or not
+     */
+    isValidated?: boolean;
```

Comment:
When would this not be validated? I don't quite understand the flow and the purpose of this.

Reply:
Discussed offline about why we want to do this.